python/agents/customer-service/customer_service/tools/tools.py

```
@@ -371,6 +374,16 @@ def generate_qr_code(
         >>> generate_qr_code(customer_id='123', discount_value=10.0, discount_type='percentage', expiration_days=30)
         {'status': 'success', 'qr_code_data': 'MOCK_QR_CODE_DATA', 'expiration_date': '2024-08-24'}
     """
+    
+    # Guardrails to validate the amount of discount is acceptable for a auto-approved discount.
+    # Defense-in-depth to prevent malicious prompts that could circumvent system instructions and
+    # be able to get arbitrary discounts.
+    if discount_type == "" or discount_type == "percentage":
+        if discount_value > 10:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)